cadcutils/cadcutils/old_get_cert.py

```diff
@@ -127,9 +131,24 @@ def _signal_handler(signal, frame):
                         help="Deprecated. Used for testing only")
 
     args = parser.parse_args()
+    if args.cert_server:
+        # update service_realm
+        dummy = BaseWsClient(CRED_RESOURCE_ID, Subject(),
+                             agent="getCert/1.0", retry=True,
+                             host=args.cert_server)
+        access_url = dummy._get_url((SERVICE_AVAILABILITY_ID, None))
+        service_realm = urlparse(access_url).netloc
+        print('Service realm: {}'.format(service_realm))
```

Comment:
Do you want the print statement to remain?

Reply:
I think it would be useful since it could be different from the one stated in the help.